int8 pwm_duty_hip
int8 pwm_duty_knee

